### ISSUE TITLE
feature: optionally insert new lines between import blocks

### ISSUE THREAD

**Issue by gavinhenderson** [trivago/prettier-plugin-sort-imports#1]
This project is amazing! I am forever fixing imports to be in an order like this.

The only thing that's stopping me using it is that I like to have a new line between the import blocks. So something like:

```
import react from 'react';

import util from '@app/util';

import relativeImport from '../../thing';
```

I'm sure lots of people don't want this so maybe it makes sense to have it be an optional flag?

**Comment by ayusharma** [comment]
Thanks for the suggestion @gavinhenderson . I'll give a try to implement it. 😄

**Comment by ayusharma** [comment]
Hi, @gavinhenderson the requested feature has released in `1.1.0`

**Comment by gavinhenderson** [comment]
@ayusharma Thanks! Just put this into the codebase over at [findmypast](https://github.com/findmypast). Much cleaner imports now 👌
